server/ServerApp.cpp

```diff
@@ -440,6 +441,33 @@ void ServerApp::HandleShutdownMessage(const Message& msg, PlayerConnectionPtr pl
     m_fsm->process_event(ShutdownServer());
 }
 
+void ServerApp::HandleLoggerConfig(const Message& msg, PlayerConnectionPtr player_connection) {
+    int player_id = player_connection->PlayerID();
+    bool is_host = m_networking.PlayerIsHost(player_id);
+    if (!is_host && m_networking.HostPlayerID() != Networking::INVALID_PLAYER_ID) {
+        WarnLogger() << "ServerApp::HandleLoggerConfig rejecting message from non-host player id = " << player_id;
+        return;
+    }
+
+    DebugLogger() << "Handling logging config message from the host.";
+    std::set<std::tuple<std::string, std::string, LogLevel>> options;
+    ExtractLoggerConfigMessageData(msg, options);
+
+    SetLoggerThresholds(options);
+
+    // Forward the message to all the AIs
+    const auto relay_options_message = LoggerConfigMessage(Networking::INVALID_PLAYER_ID, options);
+    for (ServerNetworking::established_iterator players_it = m_networking.established_begin();
+         players_it != m_networking.established_end(); ++players_it)
+    {
+        if ((*players_it)->GetClientType() == Networking::CLIENT_TYPE_AI_PLAYER) {
+            DebugLogger() << "Forwarding logging thresholds to AI " << (*players_it)->PlayerID();
```

Comment:
my concern was just the extra spam in the AI logs with irrelevant other-process logging level changes